test/tests/convergence/tests

```diff
@@ -0,0 +1,17 @@
+[Tests]
+  design = 'Convergence.md'
+  issues = '#25931'
+  [nonlinear]
+    type = 'Exodiff'
+    input = 'diffusion_convergence.i'
+    exodiff = 'diffusion_convergence_out.e'
+    requirement = 'The system shall query the convergence status of the nonlinear solvers.'
+  []
+  [reference_residual]
+    type = 'Exodiff'
+    input = 'reference_residual.i'
+    exodiff = 'reference_residual_out.e'
+    requirement = 'The system shall have the ability to base convergence on the comparison of '
```

Comment:
This is a duplicate of [this requirement](https://github.com/idaholab/moose/blob/c36c4a18fba0ccf92769bbaef5e6b2d2ba7a43c1/test/tests/problems/reference_residual_problem/tests#L8). Please adjust to be unique to this test and new System.

Reply:
It is indeed, since it's supposed to replace it.